pkg/container-suseconnect/configuration.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package container_suseconnect
```

Comment:
Instead of having `pkg/container-suseconnect`, I'd simply have everything under `internal` (as hinted by @cyphar):

1. It's really unlikely that this program will grow so much more that it'd need a package restructuring.
2. Nitpicking: I believe there's a lint rule out there forbidding packages with underscores :yum:

Reply:
Alright, I changed everything you mentioned :+1: